### ISSUE TITLE
Research/Design: journey map to capture E2E experience

### ISSUE THREAD

**Issue by aplatt-coforma** [department-of-veterans-affairs/va.gov-team#66431]
### Goals
Create journey map that can be used to capture end to end experience. We can add in as we learn.

[selection -> outreach -> submission]

### To Do
- [x] Study existing Mural boards
- [ ] Provide a high-level flow / use cases for veterans to achieve benefits delivery. This can help us identify what's existing and what needs to be put in place.
- [ ] 

### DoD
- Serve as a document that can support teams to align on the experience we are proposing
- Demo post-standup or async -> get feedback from necessary parties -> generate follow up tasks if any -> close ticket

**Comment by aplatt-coforma** [comment]
Gretchen has started working on this ticket. Moving to In Progress.

**Comment by gmaciolek** [comment]
Updates so far this sprint:

- Added a [user task flow of Veteran experience](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696612899202/80df54e376713147c4b515584fb06b22eff278b6?sender=u7ebefd98ade29ab3794a2544)
- Added a [communication channel decision matrix table](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696612899202/80df54e376713147c4b515584fb06b22eff278b6?sender=u7ebefd98ade29ab3794a2544) to capture considerations for selecting a format/communication channel for pilot with additional possibilities for other channels in the future
- Shared task flow with team and used it as a launching off point for deeper discussion on monitoring/analytics 
- Shared with Shannon for feedback asynch and synchronously

Next steps:
- Merge user task flow with blueprint
- Review swimlanes and reorder/remove
- Review and and make updates based on stickies/comments/feedback
- merge similar stickies and identify outstanding questions and tasks to surface to the team

**Comment by gmaciolek** [comment]
UX Blueprint has been updated. Here's a summary of key changes.

**1. Merged the user task flow with the blueprint**
The user task flow shows the ideal flow and captures alternative actions Veterans may take. Visualizing the holistic journey will help us optimize the experience regardless of the Veteran’s preferred method of learning about and filing a claim. (Ideal/happy path = file a claim digitally on VA.gov, breeze through claims processing without delays, be determined eligible, get compensation)

**2. Added more backstage actions**
Added cohort selection and sending communication as backstage actions that happen before the Veteran journey starts.
Connected user task flow to key backstage actions in the claims processing 

**3. Added a timeline section to the top** 
This allows us to document what we know about how long certain parts of the process currently take (knowing that timelines vary widely) and will support conversations about success criteria with respect to timing. (e.g. How fast is a fast turnaround time from getting a communication to taking action? Are claims for recipients of our pilot outreach moved through the processes faster than average?)

**4. Added more backed systems information:**
We could probably add more, but Christie has been doing a great job tracking all of this in her work and I didn’t want to spend too much time duplicating efforts.

**5. Organized the challenges and opportunities for the first phase into themes.** 
The team had added a lot of great thoughts and ideas here and I wanted to group them in a way that would make it easier to reference as we each work on different tasks.

**6. Added some details and questions to the [communication channel decision matrix table](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696612899202/80df54e376713147c4b515584fb06b22eff278b6?wid=0-1698345835308)**

**7. Removed Internal ownership row (for now)** 
I’m open to adding this back, but I started it before I saw the [Benefits Portfolio DACI MURAL](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1685470828833/17e7dd60051d32a73dd6011312603e93d53389f8?sender=u7ebefd98ade29ab3794a2544) and I felt like capturing the details of internal ownership for the entire journey could be more time-consuming than it was worth. Maybe it makes more sense to start by identifying the internal processes and policies that apply to our work then fill in relevant information re: internal ownership/required approvals/who needs to be informed based on that.

**Suggested next steps for the backlog:** 
- Add more details re: policies and processes to make sure we’re prepared for any required approvals far enough in advance to avoid delays to our timeline.
- Document decisions about monitoring and measurement. We’ve been using a different artifact to support that conversation, but should circle back to capture what we decide to do for the pilot.
